packages/ocaml-variants/ocaml-variants.4.12.0+options/opam

```diff
@@ -15,6 +15,10 @@ depends: [
 conflict-class: "ocaml-core-compiler"
 flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build-env: [
+  [LSAN_OPTIONS = "detect_leaks=0,exitcode=0"]
```

Comment:
Shouldn't these be added only if one of the two options packages is installed?

Reply:
Yeah but this is not possible at the moment.